raiden/network/rpc/client.py

```diff
@@ -737,9 +737,10 @@ class TransactionPending:
     extra_log_details: Dict[str, Any]
 
     def __post_init__(self) -> None:
-        log.debug("Transaction created", **self.to_log_details())
         self.extra_log_details.setdefault("token", str(uuid4()))
 
+        log.debug("Transaction created", **self.to_log_details())
```

Comment:
Hum, funny, this was done by https://github.com/raiden-network/raiden/pull/6110/files